layout.html

```diff
@@ -138,6 +138,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <!-- Owen is managing the Bing verification-->
     <meta name="msvalidate.01" content="7D866336EBF5D0BAD8327269CD856D25" />
+    
+    {% if page.slug.size > 9 %}
+      {% assign check_string = page.slug | slice: -9, 9 %}
```

Comment:
```suggestion
      {% comment %}
      This page should not be indexed because something something
      {% endcomment %}
      {% assign check_string = page.slug | slice: -9, 9 %}
```
It's not clear what's motivating this, please include a comment :slightly_smiling_face: